base/applications/calc/calc.h

```diff
@@ -1,16 +1,28 @@
-#ifndef _CALC_H
-#define _CALC_H
+#ifndef __CALC_H__
+#define __CALC_H__
 
-#include <stdarg.h>
-#include <windef.h>
-#include <winuser.h>
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
 #include <tchar.h>
+#include <commctrl.h>
+#include <shellapi.h>
+
+#include <stdio.h>
+#include <string.h>
 #include <math.h>
 #include <float.h>
 #include <malloc.h>
 #ifndef DISABLE_HTMLHELP_SUPPORT
 #include <htmlhelp.h>
 #endif
+#include <limits.h>
+
+/* RESOURCES */
+#include "resource.h"
+
+#ifndef IDC_STATIC
+#define IDC_STATIC  ((DWORD)-1)
+#endif
```

Comment:
Move it into resource.h (see comment below).

Reply:
If I do this, I'm getting signed/unsigned mismatches at compile time. Didn't you?

Reply:
`~0u`

Reply:
Check these ones:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=define+IDC_STATIC+
they work during compilation.

Reply:
They work because they are used only by resource file and not used by any other source code, perhaps.

Reply:
Delete `(DWORD)` type cast.